src/apollo/client.js

```diff
@@ -1,14 +1,27 @@
 import { ApolloClient, HttpLink, InMemoryCache } from '@apollo/client';
 
+const defaultOptions = {
+  query: {
+    errorPolicy: 'all',
+    fetchPolicy: 'no-cache',
+  },
+  watchQuery: {
+    errorPolicy: 'ignore',
+    fetchPolicy: 'no-cache',
+  },
+};
+
 export const compoundClient = new ApolloClient({
   cache: new InMemoryCache(),
+  defaultOptions,
```

Comment:
as per my comment, let's remove the defaultOptions for compoundClient only

Reply:
```suggestion
```